src/test/java/org/minions/devfund/mauricioz/linkedList/ContainsTest.java

```diff
@@ -0,0 +1,341 @@
+package org.minions.devfund.mauricioz.linkedList;
+
+import java.util.LinkedList;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ Source code recreated from a .class file by IntelliJ IDEA
+ (powered by Fernflower decompiler)
+ */
+public class ContainsTest {
+    public ContainsTest() {
+    }
+
+    /**
+     * setup for tests.
+     * @throws Exception
+     */
+    @Before
+    public void setUp() throws Exception {
+    }
+
+    /**
+     * test.
+     */
+    @Test
+    public void testContainsMiddle() {
+        LinkedList var1 = new LinkedList();
+        LinkedList var2 = new LinkedList();
+        var1.add(4);
+        var1.addLast(8);
+        var1.addLast(3);
+        var1.addLast(12);
+        var1.addLast(6);
+        var2.add(8);
+        var2.add(3);
+        var2.add(12);
+
+        try {
+            boolean var3 = LinkedListUtils.containsSubsequence(var1, var2);
+            Assert.assertTrue("containsSubsequence returns false when first LinkedList contains second in middle of list", var3);
+        } catch (Exception var4) {
+            Assert.fail("containsSubsequence throws " + var4 + " when first LinkedList contains second in middle of list");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define a constant instead of duplicating this literal "containsSubsequence throws " 12 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)